### ISSUE TITLE
custom property support

### ISSUE THREAD

**Issue by VinSpee** [stephenway/postcss.vim#1]
Any plan to add support for custom properties? ex:

```
--MyStyle: var(--Other-Style);
```

**Comment by stephenway** [comment]
Yes I plan to add in support for all the things, but haven't had enough time to get back to this. Actually forgot to add to the readme that it's a work in progress, not full featured yet.
